### PR TITLE
Frame exporter

### PR DESCRIPTION
**Pull request by fredrikiselius**
Added functions to export a interval of frames.
The interval is set with two buttons in the videowidget and the export dialog can be found under tools->export->frames.

### CODE REVIEW

**Comment by henpe071** [comment]
I have not looked at the code yet, but i knew directly that i could crash this one. If you want to do things from the menu you need checks. Now if you start the program and click tools->export->frames it crashes.

I've fixed the other similar crashes today, that's why I guessed you had missed this one as well

**Comment by fredrikiselius** [comment]
No it doesn't. Check the code ;)

**Review by henpe071 — COMMENTED** [review]
First, yes it crashes.
Next, When you press the first(left) bracket for the first time it updates the right one, sure now you get an interval from 1(?; not 0 the first frame?) to the current frame but the left button should only update the left bracket. It also pushes the interval to the right if you click on a frame to the right of the right bracket again not what you want because it moves both brackets. 
You need to decide if the buttons should update only their respective bracket or as they do now where you update to get a legal interval. If you choose as it is now, why even have two buttons?
Now if i press left button on frame 10 i get interval 1-10, then press right button on frame 20 i get interval 1-20. Seems wrong.

A way to see the interval without having to set a new one or go up in the menu would be nice.
As would some tooltips on the right buttons(left have).
Maybe even add some shortcuts.
You might want a way to delete the interval though it isn't as necessary.
When you change video you wanna reset the interval. A marked interval in one video should not be the same in another one. At least you kinda check the bounaries so you can't get a frame outside of the video but that shouldn't be as needed if you reset it. 
When you export the images, maybe create a map to put them in instead of just dumping them.
PUPUP
Please remove the ?-sign from the popup window.
If you don't choose a path they dont seem to save. (?)
Make it possible to export only one frame.
Right now "from-to" scrollbox auto updates, sure thats fine, but if i say i wanna export frame 1-1 (so only frame 1) i get frame 0-1 or 1-2 depending on which box i was in last. Also is ot possible to update the boxes as soon as you click outside the box without haveing to click another widget?
It would be nice to emit something to the status bar when it's done exporting.
If you enter a path that doesn't exist you get a popup window asking if it should create it. Good, but if you press no both popup closes when only the new one asking yes/no should. Not the export frame popup.

EDIT: when you enter a path, if you write just "alala" you get a map, maybe not exactly where the user guess but at least you get one. If you write something the program doesn't like you should get some kind or error/popup that says the map can't be created. At least i can't find my map named "".

I'll start to look through the code soon.

File: ViAn/GUI/videowidget.h [hunk 1]
```diff
@@ -87,6 +89,11 @@ public slots:
     void update_bar_pos(int change_x, int change_y);
     void set_current_frame_size(QSize size);
     void on_bookmark_clicked(void);
+    //void next_poi_clicked(void);
+    //void prev_poi_clicked(void);
```
Comment (henpe071) [inline ViAn/GUI/videowidget.h]: remove

**Review by henpe071 — APPROVED** [review]
Good!!

**Pull request merged**